solr/modules/sql/src/test/org/apache/solr/handler/sql/TestSQLHandler.java

```diff
@@ -425,22 +424,22 @@ public void testBasicSelect() throws Exception {
 
     tuples = getTuples(sParams, baseUrl);
 
-    assert (tuples.size() == 3);
+    assertEquals(3, tuples.size());
 
     tuple = tuples.get(0);
-    assert (tuple.getLong("myId") == 3);
-    assert (tuple.getLong("myInt") == 20);
-    assert (tuple.get("myString").equals("a"));
+    assertEquals(3, (long) tuple.getLong("myId"));
```

Comment:
I think so - I'll take a look

Reply:
Hmmm so `tuple.getLong` returns a `Long` because it also returns `null` sometimes. Changing to `long` can't be done for `tuple.getLong` since then `null` can't be returned. `3L` is still a `long` and can't be compared with `Long` apparently. I could change it to

```
assertEquals(Long.valueOf(3), tuple.getLong("myId"))
```

but that seems worse.

I don't have any other ideas right now :/

Reply:
@mkhludnev shared `assertEquals(3, tuple.getLong("myId").longValue());` which is even better than what I could come up with.

Reply:
addressed in 6492b9a68252f92db11fbee074acbbbf05680a51 for all the `getLong` and some other redundant casts I found along the way.